python/lsst/eotest/sensor/EOTestPlots.py

```diff
@@ -1067,51 +1067,58 @@ def _ingestResults(self, results_file, xtalk_file=None):
         self['CCD-014'].ok = (dark_current < 0.2)
 
-        bands = self.plotter.qe_data['QE_BANDS'].data.field('BAND')
-        bands = OrderedDict([(band, []) for band in bands])
-        for amp in imutils.allAmps():
-            try:
-                values = self.plotter.qe_data['QE_BANDS'].data.field('AMP%02i' % amp)
-                for band, value in zip(bands, values):
-                    bands[band].append(value)
-            except KeyError:
-                pass
-        for band, specnum, minQE in zip('ugrizy', list(range(21, 27)),
-                                        (41, 78, 83, 82, 75, 21)):
-            try:
-                qe_mean = np.mean(bands[band])
-                self['CCD-0%i' % specnum].measurement = '%.1f\\%%' % qe_mean
-                self['CCD-0%i' % specnum].ok = (qe_mean > minQE)
-            except KeyError:
-                self['CCD-0%i' % specnum].measurement = 'No data'
-        prnu_results = self.plotter.prnu_results
-        target_wls = set(EOTestPlots.prnu_wls)
-        ratios = {}
-        for wl, stdev, mean in zip(prnu_results['WAVELENGTH'],
-                                   prnu_results['STDEV'], prnu_results['MEAN']):
-            if stdev > 0:
-                target_wls.remove(int(wl))
-                ratios[wl] = stdev/mean
-                if wl in self.prnu_specs:
-                    if ratios[wl] < 0.01:
-                        self.prnu_specs[wl].measurement = \
-                            "\\num{%.1e}\\%%" % (ratios[wl]*100)
-                    else:
-                        self.prnu_specs[wl].measurement = \
-                            "%.2f\\%%" % (ratios[wl]*100)
-                    self.prnu_specs[wl].ok = (ratios[wl] < 5e-2)
-        max_ratio = max(ratios.values())
-        max_wl = list(ratios.keys())[np.where(list(ratios.values()) == max_ratio)[0][0]]
-        if max_ratio < 0.01:
-            self['CCD-027'].measurement = 'max. variation = \\num{%.1e}\\%% at %i\\,nm' % (
-                max_ratio*100, max_wl)
-        else:
-            self['CCD-027'].measurement = 'max. variation = %.2f\\%% at %i\\,nm' % (max_ratio*100, max_wl)
-        if target_wls:
-            measurement = self['CCD-027'].measurement + '\\\\missing wavelengths: ' \
-                + ', '.join([str(x) for x in sorted(target_wls)]) + "\\,nm"
-            self['CCD-027'].measurement = '\\twolinecell{%s}' % measurement
-        self['CCD-027'].ok = (max_ratio < 5e-2)
+        ## The following code is commented out because as of 2017/12/15 we are not producing
+        ## the necessary data to produce these plots. It has not been *removed* as it is possible
+        ## that this might be added in future. A config option to switch this is not appropraite
+        ## as this is a 3rd party package, so this commenting out just lives on its own commit
+        ## so that it can a) easily be undone, and b) won't be merged upsteam.
+        ##############################
```

Comment:
I would recommend using triple quotes to comment out the entire block. This makes fewer changes in the diff (both now, and if they are re-added in the future) and is easier to undo.

Also, omit line 1075 and replace with a newline.